juju/juju
# Firewaller worker on offering model opens ingress holes in firewall

## Description of change

Followup from https://github.com/juju/juju/pull/7540
The firewaller listens for required ingress changes on the offering side of a cross model relation and opens the relevant ports.
A new collection is used to store incoming ingress changes from the consuming side. A new watcher notifies the firewaller worker of those changes.

## QA steps

Deploy a cross model scenario on AWS. Use the console to see that the security group has the right ports opened.
